docs/babel-macros.md

```diff
@@ -9,7 +9,7 @@ Most of Emotion's Babel Macros are available by adding `/macro` to the end of th
 ```jsx
 import styled from '@emotion/styled/macro'
 import css from '@emotion/css/macro'
-import { css, keyframes, injectGlobal } from 'emotion/macro'
```

Comment:
This was correct. We have 2 "versions" of Emotion:
- `@emotion/core` + `@emotion/styled`, those two are specific to React and don't work without it
- `emotion`, this is so-called "vanilla" Emotion which can be used in any app, it's completely framework-independent

So all 3 imports here are correct, even though they don't add up to a valid JavaScript when put into a single code snippet because some identifiers (such as `css`) are duplicated here.